### PR TITLE
Dockerfile.ubi: Work around krb5-libs CVE until the UBI image is refreshed

### PR DESCRIPTION
**Pull request by jhrozek**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
The current version of Dockerfile.ubi is vulnerable against
CVE-2022-42898, according to
https://catalog.redhat.com/software/containers/ubi8-minimal/
and also a quick local test.

Until an updated image is released, let's upgrade the version directly in
the Dockerfile to make security scanners happy.

#### Which issue(s) this PR fixes:
None

#### Does this PR have test?
Implicit through GH actions

#### Special notes for your reviewer:
I haven't linked issue #1340 explicitly because we still want the image
to be bumped.

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: Dockerfile.ubi [hunk 1]
```diff
@@ -37,10 +37,13 @@ ARG STATIC_LINK=no
 RUN make
 
 # hash below referred to latest
-FROM registry.access.redhat.com/ubi8/ubi-minimal@sha256:33931dce809712888d1a8061bfa676963f517daca993984afed3251bc1fb5987
+FROM registry.access.redhat.com/ubi8-minimal@sha256:4ed971a5564d2cfedf750793da53ef6f1fdf295263f1cc31d703aa8acf6d02cf
```
Comment (saschagrunert) [inline Dockerfile.ubi:40]: Can we add this and the `go-toolset` image above to `dependencies.yaml`?

**Comment by saschagrunert** [comment]
Prow seems to be down, there are no jobs running and no labels applied to this PR.

File: Dockerfile.ubi (hunk 1 above)
Comment (jhrozek) [reply Dockerfile.ubi]: good idea, done

Comment (jhrozek) [reply Dockerfile.ubi]: oh, I haven't added go-toolset. On it.

Comment (jhrozek) [reply Dockerfile.ubi]: go-toolset added

**Review by saschagrunert — APPROVED** [review]

**Comment by saschagrunert** [comment]
Thanks!

**Pull request merged**